src/components/AddingForm/AddingForm.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  FC,
+  useState,
+  useEffect,
+  Dispatch,
+  SetStateAction,
+  ChangeEvent,
+  FormEvent,
+  useRef,
+} from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Errors } from '../../types/Errors';
+
+type Props = {
+  areAnyTodos: boolean,
+  userId: number,
+  isInputDisabled: boolean,
+  allCompleted: boolean,
+  onSubmit: (todoData: Omit<Todo, 'id'>) => Promise<void>,
```

Comment:
Consider using 'Pick<Todo, 'title' | 'completed' | 'userId'>' instead of 'Omit<Todo, 'id'>'. It explicitly lists the fields you need and can be more readable 😊